symfony/src/Controller/CourseOverviewController.php

```diff
@@ -17,7 +17,7 @@
 class CourseOverviewController extends AbstractController
```

Comment:
Controllers are controlling (pun intended) routes for specific entities. By creating a `CourseOverviewController` and a `CourseSelection` (the controller suffix is missing btw), we are mixing up the use cases of rendering pages and querying data through routes. For routes querying data specific to the entity `Kurs` there must not be more than a single `KursController`.

Reply:
Just talked this one over with midnightthings, my take on it isn't exclusively right, feel free to ignore it.